test/image/mocks/zz-subtitle.json

```diff
@@ -0,0 +1,27 @@
+{
+    "data": [
+        {
+            "x": [1, 2, 3, 4],
+            "y": [10, 12, 11, 13],
+            "type": "scatter"
+        }
+    ],
+    "layout": {
+        "title": {
+            "text": "A simple plot",
```

Comment:
The following case is also working nicely now.
Let's add a mock for it.
```json
        "title": {
            "text": "Main title<br>2nd line",
            "subtitle": {
                "font": {
                    "weight": "bold",
                    "style": "italic",
                    "variant": "small-caps",
                    "textcase": "word caps",
                    "lineposition": "under",
                    "shadow": "2px 2px 4px yellow"
                },
                "text": "subtitle<br>2nd line"
            }
        }
```

Reply:
Added! https://github.com/plotly/plotly.js/pull/7012/commits/e2fff2650c940711fec7304ac1af73bed623f668